vite.config.js

```diff
@@ -0,0 +1,55 @@
+import { resolve } from "path";
+import { defineConfig } from "vite";
+import { VitePWA } from "vite-plugin-pwa";
+
+export default defineConfig({
+  resolve: {
+    alias: {
+      "@fonts": resolve(__dirname, "assets/fonts"),
+    },
+  },
+  build: {
+    outDir: "build",
```

Comment:
```suggestion
    outDir: "build",
    emptyOutDir: false,
```
might be an option eventually (and removing the config file from public)?

Reply:
It should be removed from public but I would not recommend to keep state in the output directory.